gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-api/src/main/idl/org.apache.gobblin.service.flowconfigsV2.restspec.json

```diff
@@ -79,7 +79,12 @@
       } ]
     } ],
     "entity" : {
-      "path" : "/flowconfigsV2/{id}"
+      "path" : "/flowconfigsV2/{id}",
+      "actions" : [ {
+        "name" : "triggerExecution",
```

Comment:
The action name is a bit strange. What options do we have here? run/start/execute/startFlow/startExecution ... ?

"trigger" is an uncommon way to describe an action in APIs.

Reply:
Is it? I can think of a couple LI tools that use "trigger" to describe an API action, and for example Azkaban uses "trigger" when referring to flows launched through data landing or kafka events: https://azkaban.readthedocs.io/en/latest/eventTrigger.html

To me, trigger seems like a good way to describe launching a flow that exists through a one-time request instead of due to the schedule or because the flowConfig was just sent.

Reply:
Looks like Azkaban use "trigger" as a noun, and there are noun-triggers in database world as well. 
Google translate says that as a verb, trigger means "cause (an event or situation) to happen or exist." . I think it's more common to use this verb to describe something in the past, like "flow was triggered by a schedule".

https://translate.google.com/?sl=auto&tl=en&text=trigger&op=translate

Also, this action will set "runImmediately:true", so we might just call it run or runImmediately for consistency.

Reply:
Oh changed it to `executeFlow`, but just noticed you suggested `runImmediately`, I think that makes sense, I will change it to that.